source/routing/routing.cpp

```diff
@@ -73,18 +73,28 @@ std::string PathItem::print() const {
 }
 
 
-bool use_crow_fly(const type::EntryPoint& point, const type::StopPoint* stop_point, const type::Data& data){
+bool use_crow_fly(const type::EntryPoint& point, const type::StopPoint* stop_point,
```

Comment:
not sure, but maybe it would be nice to add a block comment to explain the whole mechanism